lib/Client/ClientTracer.php

```diff
@@ -174,26 +179,16 @@ private function _initThriftDataIfNeeded($componentName, $accessToken) {
             'lightstep.tracer_version'  => LIGHTSTEP_VERSION,
         ];
 
-        // Generate the GUID on thrift initialization as the GUID should be
+        // Generate the GUID on initialization as the GUID should be
         // stable for a particular access token / component name combo.
         $this->_guid = $this->_generateStableUUID($accessToken, $componentName);
-        $this->_thriftAuth = new \CroutonThrift\Auth([
-            'access_token' => strval($accessToken),
-        ]);
+        $this->_auth = new Auth($accessToken);
 
-        $thriftAttrs = [];
+        $attrs = [];
         foreach ($runtimeAttrs as $key => $value) {
-            array_push($thriftAttrs, new \CroutonThrift\KeyValue([
-                'Key' => strval($key),
-                'Value' => strval($value),
-            ]));
-        }
-        $this->_thriftRuntime = new \CroutonThrift\Runtime([
-            'guid' => strval($this->_guid),
-            'start_micros' => intval($this->_startTime),
-            'group_name' => strval($componentName),
-            'attrs' => $thriftAttrs,
-        ]);
+            array_push($attrs, new KeyValue(strval($key), strval($value)));
```

Comment:
Would be better to use the syntax
```
$attrs[] = new KeyValue(strval($key), strval($value)));
```
since there is no function call overhead.
See: http://php.net/manual/en/function.array-push.php

Reply:
that's very disconcerting, done